src/components/seo.js

```diff
@@ -24,35 +24,43 @@ function SEO({ description, lang, meta, keywords, title }) {
       }}
       meta={[
         {
-          name: `description`,
+          name: `description',
           content: metaDescription,
         },
+        {
+          property: 'og:title',
+          content: 'Mentor Night | Free Code Camp',
```

Comment:
Like `metaDescription`, it might be better to derive the title from gatsby-config.js using site.siteMetadata.title. We would need another variable below line 18, in that case.